src/test/java/com/google/api/generator/gapic/composer/grpc/GrpcServiceCallableFactoryClassComposerTest.java

```diff
@@ -30,6 +30,7 @@ public void generateServiceClasses() {
 
     Assert.assertGoldenClass(this.getClass(), clazz, "GrpcEchoCallableFactory.golden");
     Assert.assertEmptySamples(clazz.samples());
+    Assert.assertCodeEquals(clazz.defaultHost(), "");
```

Comment:
I don't think we need the assertions in this tests, because we didn't touch `AbstractServiceCallableFactoryClassComposer` or `GrpcServiceCallableFactoryClassComposer`.

Reply:
Thanks that makes sense - I forgot to update after I refactored. Updated!